api/PromiseRejectionEvent.json

```diff
@@ -42,16 +42,16 @@
             "version_added": false
           },
           "opera": {
-            "version_added": false
+            "version_added": "36"
           },
           "opera_android": {
             "version_added": false
           },
           "safari": {
-            "version_added": false
+            "version_added": "11"
           },
           "safari_ios": {
-            "version_added": false
+            "version_added": "11.4"
```

Comment:
On the basis of https://bugs.webkit.org/show_bug.cgi?id=150358 I trust that these Safari changes actually shipped, but I couldn't place the version numbers and 11.4 isn't in our Safari for iOS data at all. I'm willing to let the desktop version be, but I'm inclined to suggest that we go with `null` for iOS, if we don't have a more direct source for this version number. `null` would also satisfy the linter.

Reply:
Based upon my research in #2006, I found that iOS 11.3 and 11.4 use Safari 11.1.  Since iOS 11.3 is the version where Safari is updated, we should set this one to "11.3".  At least, once #2006 has been completed.  :wink: